community/boilerplates/nextjs-postgres-graphql/README.md

```diff
@@ -0,0 +1,107 @@
+# nextjs-postgres-graphql
+
+Boilerplate to get started with Nextjs, Hasura GraphQL engine as CMS and postgres as database using this awesome library: [withData](https://github.com/adamsoffer/next-apollo).
+
+![Nextjs Postgres GraphQL](./assets/nextjs-postgres-graphql.png)
+
+# Tutorial
+
+- Deploy Postgres and GraphQL Engine on Heroku:
+  
+  [![Deploy to
+  heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+  Please checkout our [docs](https://docs.hasura.io/1.0/graphql/manual/deployment/index.html) for other deployment methods
+
+- Get the Heroku app URL (say `my-app.herokuapp.com`)
+- Create `author` table:
+  
+  Open Hasura console: visit https://my-app.herokuapp.com on a browser  
+  Navigate to `Data` section in the top nav bar and create a table as follows:
+
+  ![Create author table](./assets/add_table.jpg)
```

Comment:
@karthikvt26 I see 6 images added to assets, but only 5 have been used in the readme. 
Also, aren't the images duplicates of the ones present at https://github.com/hasura/graphql-engine/tree/master/community/boilerplates/gatsby-postgres-graphql/assets ?

Can we link to them instead of adding those again?

Reply:
Fixed.